### PR TITLE
Update NU5131.md

### PR DESCRIPTION
**Pull request by zivkan**
Fixes: #1732

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[335870d](https://github.com/NuGet/docs.microsoft.com-nuget/commits/335870df2829df2cbdc79ea7fe3acfccc035d711)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5131.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-patch-1/docs/reference/errors-and-warnings/NU5131.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5131?branch=pr-en-us-1733) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910101549114014-1733%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mJnI97qnZwOYa5AnrZID4AA9FkFSDUA2nkXroGKexFU%3D&st=2019-10-10T15%3A45%3A13Z&se=2019-11-10T15%3A50%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5131.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The package author wants to prevent package consumers from writing code that dir
 </references>
 ```
 
-This works for projects using the package with `packages.config`, but to work as intended with `ProjectReference`, the project must also contain the following files:
+This works for projects using the package with `packages.config`, but to work as intended with `PackageReference`, the package must also contain the following files:
```
Comment (loic-sharma) [inline docs/reference/errors-and-warnings/NU5131.md]: What do you think of something like this?

```suggestion
This package will not work as intended when using `PackageReference`. To fix this, the package must also contain the following files:
```

Comment (loic-sharma) [reply docs/reference/errors-and-warnings/NU5131.md]: Is there a link that explains what `PackageReference` is? If so, please add links

**Comment by loic-sharma** [comment]
Can we suggest that the customer try packing using `nuget.exe pack` or `dotnet pack` instead?

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[95dfccb](https://github.com/NuGet/docs.microsoft.com-nuget/commits/95dfccb8fc30cdac20aef7974c05f1a41c1fa47b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU5131.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-patch-1/docs/reference/errors-and-warnings/NU5131.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5131?branch=pr-en-us-1733) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910152113184280-1733%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xV1NUTJQnWEt18xP8ARTesCcudI%2FG5%2Bd3tjwxbpX%2FWE%3D&st=2019-10-15T21%3A09%3A20Z&se=2019-11-15T21%3A14%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**